docs/advanced_topics_performance_tuning.rst

```diff
@@ -31,7 +31,7 @@ An example:
 .. code-block:: bash
 
   num_threads=5
```

Comment:
What is this doing here? It's not set in the shell, is it?

Reply:
It is just to show the evn conditions.